src/tests/parser_test.py

```diff
@@ -562,18 +563,22 @@ def test_matrix_multiplication_with_decorators(code):
     assert inner_function.decorators[0].name == 'a'
 
 
-def test_module_publicity():
+@pytest.mark.parametrize("parent_path", (
+    Path("some").joinpath("directory"),
```

Comment:
```suggestion
    Path("some") / "directory",
```

can you also replace the others please :)

Reply:
Sure thing, I'll find some time today to work on this!

Reply:
Updated in 10e1e7aad14628d52e14788aa5a5807ee9928631.